### PR TITLE
chore: Prepare release v1.7.0

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,11 @@
     "liferay",
     "yeoman-generator"
   ],
-  "devDependencies": {
+  "engines": {
+    "node": ">= 8.0.0",
+    "npm": ">= 6.0.0"
+  },
+  "dependencies": {
```
Comment (victorg1991) [inline package.json]: We are no longer separating devDependencies from dependencies?

File: README.md [hunk 2]
```diff
@@ -122,23 +151,23 @@ imported into Liferay.
...
 
 ```bash
 npm run preview
 ```
 
-### Packaging fragments for distribution
+### Packaging for distribution
 
-After you have finished the development of fragments, they can be distributed as
+After you have finished the development of elements, they can be distributed as
```
Comment (victorg1991) [inline README.md]: of the project?

File: README.md [hunk 3]
```diff
@@ -65,48 +99,43 @@ inside by running
...
+Fragment compositions are treated as regular fragments, so they need to be grouped
+inside collections too:
 
-Collections and fragments must follow a specific (although simple) directory
-structure so that they can be imported into a Liferay instance. The information
-about each collection and fragment is stored inside `JSON` files, and you can
-change them manually.
+```bash
+npm run add-fragment-collection
+```
```
Comment (victorg1991) [inline README.md:109]: Here we need to add the command for creating composition, right?

Comment (victorg1991) [reply README.md:109]:
```bash
npm run add-fragment-composition
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -30,6 +30,20 @@ All CLI features must be _compatible_ with all Liferay Portal versions: if
 they depend on some portal feature (API call ,service), they should fail
 gracefully and so a nice error, but never break anything.
 
+### Creating a new release
+
+1. Create a new branch and add changes if needed:
+   - `docs: x` update outdated documentation.
+   - `chore: Update/Remove x` update dependencies (remove unused, upgrade if possible).
```
Comment (p2kmgcl) [inline CONTRIBUTING.md]: `npm audit`, `npm outdated`

**Review by sandrodw3 — APPROVED** [review]

**Pull request merged**